core/libp2p/connection/raw_connection.hpp

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_RAW_CONNECTION_HPP
+#define KAGOME_RAW_CONNECTION_HPP
+
+#include "libp2p/basic/readwritecloser.hpp"
+
+namespace libp2p::connection {
+
+  struct RawConnection : public basic::ReadWriteCloser {
```

Comment:
Why just Connection is not enough?

Reply:
We have many different connections, and here I emphasize the fact that this connection is raw (neither secured, nor multiplexed)